sdk/servicefabricmanagedclusters/Microsoft.Azure.Management.ServiceFabricManagedClusters/tests/Microsoft.Azure.Management.ServiceFabricManagedClusters.Test.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>ServiceFabricManagedClusters.Test</PackageId>
+    <Description>ServiceFabricManagedClusters Tests Class Library</Description>
+    <AssemblyName>ServiceFabricManagedClusters.Test</AssemblyName>
+    <Version>0.1.0</Version>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <ProjectReference Include="..\src\Microsoft.Azure.Management.ServiceFabricManagedClusters.csproj" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <!--Do not remove until VS Test Tools fixes #472-->
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
Why is this needed?

Reply:
Please remove this, unless there is a very good reason this is needed

Reply:
@markcowl removed thanks. I copied it over from somewhere.